src/test/java/ParserTest.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Test class to test parse() method in Parser.
+ */
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParserTest {
+    @Test
+    public void parseTest() {
```

Comment:
I would recommend using this naming standard to name your test methods: featureUnderTest_testScenario_expectedBehavior()